ImperatorToCK3/Data_Files/configurables/succession_law_map.txt

```diff
@@ -0,0 +1,39 @@
+# IMP-CK3 SUCCESSION LAW MAPPINGS #
+# link = { imp = [law] ck3 = [law] }
+# The CK3 law is output in the title's history
+# folder, right after the line defining
+# the holder, as succession_laws = { [law] }
+
+# CK3 has default succession laws
+
+# For mappings with multiple ck3 laws, all laws will be written out
+
+
+# Monarchy
+
+link = { imp = agnatic_succession_law ck3 = male_only_law ck3 = single_heir_succession_law }
+link = { imp = agnatic_succession_law ck3 = single_heir_succession_law }
```

Comment:
Same imp part as the mapping above. Is there something that distinguishes them?

Reply:
No, I left the second mapping by mistake after merging the two.